### PR TITLE
add deploy-6 docs

### PR DESCRIPTION
**Pull request by gomathyk**

### CODE REVIEW

File: en/docs/deploy/identity-provider-related-tables.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Identity Provider Related Tables
+# identity provider Related Tables
```
Comment (dewniMW) [inline en/docs/deploy/identity-provider-related-tables.md:1]: why is this change required for H1?

File: en/docs/deploy/identity-provider-related-tables.md [hunk 2]
```diff
@@ -35,12 +35,11 @@ table. Following are the columns of the table.
...
-An Identity Provider may have different roles for authorization which
-are different from the local roles of the Identity Server. In such a
-situation, roles at the Identity Provider can be mapped to the local
-roles in the Identity Server. Such roles of the Identity Provider are
-stored in this table. ROLE column contains the name of the role. `IDP\_ID`
-is the ID of the Identity Provider. Following are the columns of the
+An identity provider may have different roles for authorization which
+are different from the local roles of WSO2 Identity Server. In such a
+situation, roles at the identity provider can be mapped to the local
+roles in WSO2 Identity Server. Such roles are stored in this table. `ROLE` column contains the name of the role. `IDP\_ID`
```
Comment (dewniMW) [inline en/docs/deploy/identity-provider-related-tables.md:41]:
```suggestion
roles in WSO2 Identity Server. Such roles are stored in this table. `ROLE` column contains the name of the role. `IDP_ID`
```

File: en/docs/deploy/identity-related-tables.md [hunk 3]
```diff
@@ -1,22 +1,21 @@
...
-This section lists out all the identity related tables and their
-attributes in the WSO2 Identity Server database.
+This section lists out all the identity related tables and their attributes in the WSO2 Identity Server database.
 
 ---
 
 #### IDN_BASE_TABLE
 
 This table is used to provide information related to the server setup.
 It has only one column `(PRODUCT_NAME)`, which contains a row with the
```
Comment (dewniMW) [inline en/docs/deploy/identity-related-tables.md:10]: `(PRODUCT_NAME)` -> should this be enclosed within brackets?

File: en/docs/deploy/identity-related-tables.md [hunk 4]
```diff
@@ -435,9 +434,9 @@ description of the values it contains.
 
 #### IDN_ASSOCIATED_ID
 
-In the Identity Server Dashboard, users can associate their social
+In WSO2 Identity Server Dashboard, users can associate their social
```
Comment (dewniMW) [inline en/docs/deploy/identity-related-tables.md:437]: `Dashboard` is outdated

File: en/docs/deploy/service-provider-related-tables.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# Service Provider Related Tables
+# service provider Related Tables
```
Comment (dewniMW) [inline en/docs/deploy/service-provider-related-tables.md:1]: why is this change required

File: en/docs/deploy/service-provider-related-tables.md [hunk 6]
```diff
@@ -7,8 +7,8 @@ attributes in the WSO2 Identity Server database.
 
 ## SP_APP
 
-When a Service Provider is added, the details are stored in this table.
-The APP_NAME column represents the Service Provider name. Following are
+When a service provider is added, the details are stored in this table.
+The APP_NAME column represents the service provider name. Following are
```
Comment (dewniMW) [inline en/docs/deploy/service-provider-related-tables.md:11]:
```suggestion
The `APP_NAME` column represents the service provider name. Following are
```

File: en/docs/deploy/service-provider-related-tables.md [hunk 7]
```diff
@@ -51,14 +51,12 @@ column `PROP_NAME` and the value is a random integer stored in
...
 
-The Service Providers can define how to authenticate users at the
-Identity Server, for authentication requests initiated by it. While
-doing that, each Service Provider can define multiple steps and for each
-step it can pick more than one authenticator. This is called as
-Multi-level (multi-factor) Authentication. This table stores each
-authentication step added to the Service Provider. Service Provider is
+The service providers can define how to authenticate users at WSO2 Identity Server, for authentication requests initiated by it. While
+doing that, each service provider can define multiple steps and for each
+step, it can pick more than one authenticator. This is called Multi-level (multi-factor) Authentication. This table stores each
```
Comment (dewniMW) [inline en/docs/deploy/service-provider-related-tables.md:56]:
```suggestion
step, it can pick more than one authenticator. This is called multi-level (multi-factor) Authentication. This table stores each
```

File: en/docs/deploy/service-provider-related-tables.md [hunk 8]
```diff
@@ -85,10 +83,10 @@ the `SP_AUTH_STEP` table. Following are the columns of the table.
 
 ## SP_CLAIM_MAPPING
 
-When the claims of the Identity Provider are different from the Service
+When the claims of the identity provider are different from the Service
```
Comment (dewniMW) [inline en/docs/deploy/service-provider-related-tables.md:86]:
```suggestion
When the claims of the identity provider are different from the service
```

File: en/docs/deploy/service-provider-related-tables.md [hunk 9]
```diff
@@ -85,10 +83,10 @@ the `SP_AUTH_STEP` table. Following are the columns of the table.
...
 Provider, corresponding claims can be mapped from this table for each
```
Comment (dewniMW) [inline en/docs/deploy/service-provider-related-tables.md:87]:
```suggestion
provider, corresponding claims can be mapped from this table for each
```

File: en/docs/deploy/user-management-related-tables.md [hunk 10]
```diff
@@ -62,7 +61,7 @@ Following are the columns of the table.
 
 ## UM_MODULE
 
-This table is not being used in Identity Server latest version.
+This table is not being used in the latest version of WSO2 Identity Server.
```
Comment (dewniMW) [inline en/docs/deploy/user-management-related-tables.md:64]: can't we remove this info from the docs?

File: en/docs/deploy/user-management-related-tables.md [hunk 11]
```diff
@@ -275,10 +272,8 @@ columns of the table.
 
 ## UM_SYSTEM_ROLE
 
-System roles of the Identity Server are stored in this table. The
-`wso2.anonymous.role` is created by default which is a special role that
-represents a user that has not logged into the Identity Server
-Management Console. Granting "Read" access to resources for this role
+System roles of WSO2 Identity Server are stored in this table. The `wso2.anonymous.role` is created by default which is a special role that
+represents a user that has not logged in to WSO2 Identity Server Management Console. Granting Read access to resources for this role
```
Comment (dewniMW) [inline en/docs/deploy/user-management-related-tables.md:276]:
```suggestion
represents a user that has not logged in to WSO2 Identity Server Management Console. Granting read access to resources for this role
```

File: en/docs/deploy/identity-related-tables.md (hunk 4 above)
Comment (gomathyk) [reply en/docs/deploy/identity-related-tables.md:437]: Commented it out.

File: en/docs/deploy/user-management-related-tables.md (hunk 10 above)
Comment (gomathyk) [reply en/docs/deploy/user-management-related-tables.md:64]: IMO, we should let this be till the latest document is tested and restructured.

File: en/docs/deploy/identity-related-tables.md (hunk 4 above)
Comment (dewniMW) [reply en/docs/deploy/identity-related-tables.md:437]: ideally, this shouldn't be commented out. This functionality has been there in the dashboard but since now we have my account as its successor please check if the functionality is there and update it here accordingly.

Comment (gomathyk) [reply en/docs/deploy/identity-related-tables.md]: You can only view the social logins that you've already used in **My Account**. Anyway, added it back and removed the dashboard part since the table still exists.

**Review by dewniMW — APPROVED** [review]
